index.d.ts

```diff
@@ -84,6 +84,21 @@ declare namespace CSSselect {
       `a === b`.
     */
     equals?: (a: Node, b: Node) => boolean;
+
+     /**
+     * is the element in hovered state?
+     */
+    isHovered?: (elem: ElementNode) => boolean;
```

Comment:
Should I name it as `getHovered` or `isHovered` is fine?
This method checks if give `elem` is in hovered state or not. So I named it as `isHovered`.

Instead if we could have used `getState` and then check `state === 'hovered'`, then `get` suits more.

Reply:
I'm okay with these — just having a subset available seems alright.